main.rb

```diff
@@ -69,7 +69,7 @@ def assets_tags
         sleep(1)
       end
 
-      raise('Assets not found!') if tries == 0
+      puts('Assets not found!') if tries == 0
```

Comment:
because we don't build frontend on tests
i think it's better to say about issue with assets not raise exception